account_move_line_sequence/views/account_move_view.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="account_move_form__ext" model="ir.ui.view">
+        <field name="model">account.move</field>
+        <field name="inherit_id" ref="account.view_move_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='line_ids']/tree//field[1]" position="before">
+                <field name="sequence" widget="handle" />
```

Comment:
This seems to suggest the answer already - that it's only used within one booking.

So then I'd move the sorting into this view as well, and leave the model alone.

We'll also not need the extra index, because the number of move lines in one entry will never exceed 1000 anyway, so indexing won't help the sorting of that.

Reply:
@ntsirintanis funny, wiget="handle" doesn't show the drag-icon like before but the actual sequence number. People can still drag on the column entries but it's a bit counter-intuitive. Any idea how to get that back? Maybe it's in the xpath inheritence or something?

12:
![image](https://github.com/user-attachments/assets/07262278-d0b6-40af-a58d-e0ae26baaf14)

16:
![image](https://github.com/user-attachments/assets/b1057067-fb8e-4731-ace4-a2c722922632)

Reply:
that's a bit strange because I do not see the id, but the drag-icon only. Will revisit

Reply:
ok, makes sense. This functionality is already there in odoo 16. So please uninstall this module, and this should fix it. I am closing this MR

Reply:
@ntsirintanis I think you meant another MR? Regarding this PR, there's no handle widget on the Journal Items tab i.e. for accounting moves, only on the Invoice Lines tab.

![image](https://github.com/user-attachments/assets/b99e7037-02ff-4059-b86c-3bcdc1b19884)